lldb/packages/Python/lldbsuite/test/dotest.py

```diff
@@ -973,8 +973,6 @@ def run_suite():
     # Iterating over all possible architecture and compiler combinations.
     os.environ["ARCH"] = configuration.arch
     os.environ["CC"] = configuration.compiler
-    if configuration.swiftCompiler:
```

Comment:
Why does this go away here while `CC` is still in the environment? Should CC also disappear? I'm just wondering because of the apparent inconsistency.

Reply:
They've already been removed upstream.